packages/react-components/src/components/utils/testUtils.tsx

```diff
@@ -2,30 +2,51 @@
 // Licensed under the MIT license.
 
 import React from 'react';
-import { mount, ReactWrapper, shallow, ShallowWrapper } from 'enzyme';
 import { LocalizationProvider, ComponentLocale, ComponentStrings } from '../../localization/LocalizationProvider';
 import { COMPONENT_LOCALE_EN_US } from '../../localization/locales';
 import { PartialDeep } from 'type-fest';
 import { _PermissionsProvider, _getPermissions, _Permissions } from '../../permissions';
+import { render } from '@testing-library/react';
+import { LiveAnnouncer } from 'react-aria-live';
 
-/**
- * @private
- */
-export const mountWithLocalization = (node: React.ReactElement, locale: ComponentLocale): ReactWrapper => {
-  return mount(node, {
-    wrappingComponent: LocalizationProvider,
-    wrappingComponentProps: { locale }
-  });
+const withLiveAnnouncerContext = (node: React.ReactElement): React.ReactElement => (
+  <LiveAnnouncer>{node}</LiveAnnouncer>
+);
+
+/** @private */
+export const renderWithLocalization = (
+  node: React.ReactElement,
+  locale: ComponentLocale
+): {
+  rerender: (node: React.ReactElement) => void;
+  container: HTMLElement;
+} => {
+  const { rerender, container } = render(
+    withLiveAnnouncerContext(<LocalizationProvider locale={locale}>{node}</LocalizationProvider>)

```

Comment:
Why do we have to wrap with LiveAnnouncer?

Reply:
Just gets rid of warnings from the `react-aria-live` package. This warning already exists in our current tests, but this removes the warning

Reply:
(we're moving off this package shortly as it does not support react18 so this was a super quick fix to the warning)